### DISCUSSION TITLE
Contribute Buttons before 20th of June 🚨

### DISCUSSION THREAD

**Discussion by Spyware007** [Spyware007/Animating-Buttons#1346] [Announcements]
Thank you so much for all the contributions! ✨
We will soon hit **200+** 🌟 , and more than **400+** buttons have been added! 🚀

### No Issues for adding new buttons will be accepted after 20th July

PS: New UI 👀
<img width="1440" alt="Screenshot 2023-06-09 at 3 50 19 AM" src="https://github.com/Spyware007/Animating-Buttons/assets/89961974/44b7f872-d338-4251-aaf2-507fed2d0b48">

**Comment by Kalivarapubindusree** [comment]
Shall we directly add new button using raising issues in that website??
will you add score for that buttons which we put in that website?? @Spyware007

**Reply by Spyware007** [reply]
I have to think about that!
